Watchman.Discord/Areas/Administration/Controllers/MutedUsersController.cs

```diff
@@ -30,27 +29,36 @@ public MutedUsersController(DirectMessagesService directMessagesService, MutingH
         [AdminCommand]
         public async Task MutedUsers(MutedUsersCommand mutedUsersCommand, Contexts contexts)
         {
-            var mutedUsersList = await _usersService.GetUsersAsync(contexts.Server).ToListAsync();
-            var (title, description, values) = this.GetMuteEmbedMessage(mutedUsersList, contexts.Server.Id);
-            await _directMessagesService.TrySendEmbedMessage(contexts.User.Id, title, description, values);
+            var mutedUsers = _usersService.GetUsersAsync(contexts.Server);
+            var (title, description, values) = await this.GetMuteEmbedMessage(mutedUsers, contexts.Server.Id);
+            if (values.Count == 0)
+            {
+                await _directMessagesService.TrySendMessage(contexts.User.Id, "Brak wyciszonych użytkowników!");
+            }
+            else
+            {
+                await _directMessagesService.TrySendEmbedMessage(contexts.User.Id, title, description, values);
+            }
         }
-        private (string title, string description, Dictionary<string, Dictionary<string, string>> values) GetMuteEmbedMessage(List<UserContext> mutedUsersList, ulong serverId)
+
+        private async Task<(string title, string description, Dictionary<string, Dictionary<string, string>> values)> GetMuteEmbedMessage(IAsyncEnumerable<UserContext> mutedUsers, ulong serverId)
         {
             var title = "Lista wyciszonych użytkowników";
             var description = "Wyciszeni użytkownicy, powody oraz data wygaśnięcia";
             var values = new Dictionary<string, Dictionary<string, string>>();
-            foreach (var mutedUser in mutedUsersList)
+            await foreach (var mutedUser in mutedUsers)
             {
                 var muteEvent = this._mutingHelper.GetNotUnmutedUserMuteEvent(serverId, mutedUser.Id);
-                if (muteEvent != null)
+                if (muteEvent == null || DateTime.Compare(muteEvent.TimeRange.End.ToLocalTime(), DateTime.Now) < 0)
                 {
-                    values.Add($"Użytkownik: {mutedUser.Name}", 
-                        new Dictionary<string, string>
-                        {
-                            {"Powód ", muteEvent.Reason},
-                            {"Data zakończenia ", TimeZoneInfo.ConvertTimeFromUtc(muteEvent.TimeRange.End, TimeZoneInfo.Local).ToString("dd.MM.yyyy HH:mm:ss", CultureInfo.InvariantCulture) }
-                        });
+                    continue;
                 }
+                values.Add($"Użytkownik: {mutedUser.Name}",
+                    new Dictionary<string, string>
+                    {
+                            {"Powód ", muteEvent.Reason},
+                            {"Data zakończenia ", muteEvent.TimeRange.End.ToLocalTimeString() }
```

Comment:
```suggestion
                            {"Powód: ", muteEvent.Reason},
                            {"Data zakończenia: ", muteEvent.TimeRange.End.ToLocalTimeString() }
```

Reply:
na pewno? po dodaniu dwukropków w tych miejscach co mówisz, te dwukropki są podwójnie:
![image](https://user-images.githubusercontent.com/35144936/89310235-b6399980-d674-11ea-899b-fe73cdc7db9e.png)

Reply:
faktycznie, teraz widzę, że już w EmbedMessagesService to jest dodawane
ale myślę, że lepiej przenieść te dwukropki tutaj, aby mieć większą dowolność separatora w przyszłości

Reply:
a i pytanie czy ta spacja jest konieczna tutaj?

Reply:
nie :) i przeniosłam separator